### DISCUSSION TITLE
Customizing the editor in Windows 11

### DISCUSSION THREAD

**Discussion by ShyamGadde** [espanso/espanso#1346] [Q&A] [answered]
Is it possible to change the default editor from notepad to vscode when using the `espanso edit` command in Windows 11?

**Comment by kpym** [comment, marked as answer]
Yes. You have to [define a global variable](https://superuser.com/questions/949560/how-do-i-set-system-environment-variables-in-windows-10) `EDITOR` to be `code`.
